ci/images/vulnerability-scan/scan-image.sh

```diff
@@ -14,7 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -o errexit
```

Comment:
I don't think it's a good idea to remove that. This flag ensures that if any command unexpectedly returns a non-0 status, the program will stop instead of ignoring the error.

If you want to ignore a return status on a specific function, you can do the following: `myfunc $MYARG || true`

Reply:
Thanks for this suggestion. But using || true will make it always return 0. I'm working on making it actually capture the return status using something like || echo $?. There seems to be some issue. Will update when its done.

Reply:
Its done now. I've tested it with a local branch and its working.